ros_installation.md

```diff
@@ -56,82 +107,77 @@ other is `packages.osrfoundation.org`. At the time of writing:
 
-ROS packages `ros-humble-ros-gz` and `ros-rolling-ros-gz` will be available
-, these packages automatically install the necessary dependencies for Gazebo Fortress.
+To install this combination of ROS and Gazebo first install ROS and then install
+either `ros-humble-ros-gz` and `ros-rolling-ros-gz` from `packages.ros.org`.
+These packages will automatically install both Gazebo Fortress and the ros-gz
+bridge.
 
 ### ROS 2 Galactic 
-ROS 2 Galactic officially supports Gazebo Edifice
-For a fully-integrated ROS system, Edifice is the recommened
-version. The way to proceed is just to use the ROS repository and do ***not***
-use the osrfoundation repository.
-
-ROS packages `ros-galactic-ros-gz` will be available pulling the necessary
-Gazebo Edifice dependencies.
+**ROS 2 Galactic is presently end of life, and we do not recommend using this
+version of ROS or Gazebo.** However, if you must this version of ROS, it officially supports
+Gazebo Edifice. To install Gazebo Edifice simply install `ros-galactic-ros-gz`
+from `packages.ros.org`. This package will automatically install both Gazebo
+Edifice and the ros-gz bridge.
 
 ### ROS 2 Foxy
-ROS 2 Foxy supports officially the Gazebo Citadel
-version. For a fully-integrated ROS system, Citadel is the recommened
-version. The way to proceed is just to use the ROS repository and do ***not***
-use the osrfoundation repository.
-
-ROS packages `ros-foxy-ros-gz` will be available pulling the necessary
-Gazebo Citadel dependencies.
+ROS 2 Foxy officially supports Gazebo Citadel, meaning the recommended version
+of Gazebo for ROS 2 Foxy is Citadel.To install Gazebo Citadel with Foxy simply
+install `ros-foxy-ros-gz` from `packages.ros.org`. This package will
+automatically install both Gazebo Citadel and the ros-gz bridge.
 
 ### ROS 1 Noetic
-ROS Noetic officially supports Gazebo Citadel
-version. For a fully-integrated ROS system, Citadel is the recommened
-version. The way to proceed is just to use the ROS repository and do ***not***
-use the osrfoundation repository.
+ROS Noetic officially supports Gazebo Citadel, meaning the recommended version
+of Gazebo for ROS Noetic is Citadel.To install Gazebo Citadel with Noetic simply
+install `ros-noetic-ros-gz` from `packages.ros.org`. This package will
+automatically install both Gazebo Citadel and the ros-gz bridge.
 
-ROS packages `ros-noetic-ros-gz` will be available pulling the necessary
-Gazebo Citadel dependencies.
-
-## Using an specific Gazebo version with ROS 2
+## Using a specific and unsupported Gazebo version with ROS 2
 <div class="warning">
-<strong>Warning:</strong> Using this option, you won't be able to use any ROS Ubuntu package
-related to Gazebo libraries from ROS deb repository.
+<strong>Warning:</strong> Only use this approach if you absolutely need to run
+a version of Gazebo that is not officially supported by your ROS distro. Using
+this approach will make it impossible to use the official ROS Ubuntu packages.
```

Comment:
It is not that someone is not going to be able to use most of the ROS Ubuntu packages, but only a few subset. those ones that are using Gazebo in their building process.

```suggestion
this approach will make it impossible to use the official ROS Ubuntu packages that depend on Gazebo.
```

Reply:
Another possible suggestion:
```suggestion
this approach will make it impossible to use some of the official ROS Ubuntu packages.
```